livy/src/main/java/org/apache/zeppelin/livy/LivySparkInterpreter.java

```diff
@@ -46,6 +46,21 @@
         new InterpreterPropertyBuilder()
             .add("zeppelin.livy.url", DEFAULT_URL, "The URL for Livy Server.")
             .add("zeppelin.livy.master", LOCAL, "Spark master uri. ex) spark://masterhost:7077")
+            .add("spark.driver.cores", "1", "Driver cores. ex) 1, 2")
```

Comment:
I'm sure if we should have default values here? Because if they are set here then that would be the value passed to Livy and then to Spark, overriding what config default Spark might have. Should we not set it if it is empty?